docs/user-guide/astro/population/plot_spiral_arm_models.py

```diff
@@ -32,8 +32,8 @@
 
 ax_cartesian.plot(vallee_spiral.bar["x"], vallee_spiral.bar["y"])
 
-ax_cartesian.set_xlabel("x (kpc)")
-ax_cartesian.set_ylabel("y (kpc)")
+ax_cartesian.set_xlabel("x [kpc]")
+ax_cartesian.set_ylabel("y [kpc]")
```

Comment:
For example change to `DEFAULT_LABEL_TEMPLATE.format(quantity="x", unit="kpc")` here...